ee/vellum_cli/__init__.py

```diff
@@ -40,7 +40,46 @@ def push(
     )
 
 
-@main.command()
+class PullGroup(ClickAliasedGroup):
+    def get_command(self, ctx, cmd_name):
+        # First try to get the command normally
+        cmd = super().get_command(ctx, cmd_name)
+        if cmd is not None:
+            return cmd
+
+        # If no command found, call our catch-all
+        return pull_module
+
+
+@main.group(invoke_without_command=True, cls=PullGroup)
+@click.pass_context
+@click.option(
+    "--include-json",
+    is_flag=True,
+    help="""Include the JSON representation of each Resource in the pull response. \
```

Comment:
We need a name for the general "thing" that users can push and pull. `Resource`? `Entity`? `Executable`? `Executable Entity`? The last one is the one we use internally

Reply:
Let's go with Resource, this is consistent with other things like Terraform https://developer.hashicorp.com/terraform/language/resources